fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CoordinatorService.java

```diff
@@ -143,6 +145,17 @@ public CompletableFuture<CreateTableResponse> createTable(CreateTableRequest req
         TableDescriptor tableDescriptor;
         try {
             tableDescriptor = TableDescriptor.fromJsonBytes(request.getTableJson());
+            // Validate reserved column names
+            List<String> reservedColumns =
+                    Arrays.asList("_change_type", "_log_offset", "_commit_timestamp");
```

Comment:
Let's do the check in `TableDescriptorValidation#validateTableDescriptor`, and make the reserved columns as static fields.